### ISSUE TITLE
Maybe include React Native in optional path or something?

### ISSUE THREAD

**Issue by guilherme6191** [ericdouglas/react-roadmap#8]
And a suggestion would be: https://egghead.io/courses/react-native-fundamentals

**Comment by ericdouglas** [comment]
@guilherme6191 React Native is a great topic to add in our optional path, but the resource that you indicated is not free.

**Comment by guilherme6191** [comment]
@ericdouglas True, my bad. In that case, I'd recommend the official docs: https://facebook.github.io/react-native/docs/tutorial.html
